tests/filecheck/mlir-conversion/llvm_test.xdsl

```diff
@@ -1,26 +1,26 @@
 // RUN: xdsl-opt -t mlir %s | filecheck %s
 
 
-module() {
+builtin.module() {
   // Type tests
   func.func() ["sym_name" = "struct_to_struct", "function_type" = !fun<[!llvm.struct<"", [!i32]>], [!llvm.struct<"", [!i32]>]>, "sym_visibility" = "private"] {
-  // CHECK: func.func private @struct_to_struct(%arg0: !llvm.struct<(i32)>) -> !llvm.struct<(i32)> {
+  // CHECK: func private @struct_to_struct(%arg0: !llvm.struct<(i32)>) -> !llvm.struct<(i32)> {
```

Comment:
This change is unrelated, right? On which LLVM version are you? The commit in the README?

Reply:
I had to write this to make the test pass again. I was on the commit in the readme (7499...)

Reply:
Moved it to a separate commit!